cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -114,6 +114,40 @@ def test_xlint_fails(self):
         err_cnt = perform_xlint(TEST_DATA_DIR, ['toy'])
         self.assertGreater(err_cnt, 0)
 
+    def test_asset_with_invalid_displayname(self):
+        '''
+        This test verifies that when an asset has '/' in its displayname they are converted to underscores `_`
+        '''
+        content_store = contentstore()
+        course_assets = set([u'_invalid_displayname_subs-esLhHcdKGWvKs.srt'], )
+
+        import_course_from_xml(
+            self.store,
+            self.user.id,
+            TEST_DATA_DIR,
+            ['course_with_invalid_displayname'],
+            static_content_store=content_store,
+            verbose=True,
+            create_if_not_present=True
+        )
+
+        # Verify the course has imported successfully
+        course = self.store.get_course(self.store.make_course_key('Edx', 'EdxTest101', '2015_summer'))
+        self.assertIsNotNone(course)
+
+        # Get & verify all assets of the course
+        assets, count = content_store.get_all_content_for_course(course.id)
+        self.assertEqual(count, 1)
+        self.assertEqual(set([asset['displayname'] for asset in assets]), course_assets)
```

Comment:
instead of comparing sets, if we know there's just one asset, just grab the first asset:

```
display_name = assets[0]['displayname']
```
or something like that.